media-libs/ilmbase/files/ilmbase-2.2.0-post-release-fixes-v20170109.patch

```diff
@@ -0,0 +1,121 @@
+diff --git a/Half/CMakeLists.txt b/Half/CMakeLists.txt
+index 6f9714d..958d1b0 100644
+--- a/Half/CMakeLists.txt
++++ b/Half/CMakeLists.txt
+@@ -1,23 +1,24 @@
+ # yue.nicholas@gmail.com
+ 
+ ADD_EXECUTABLE ( eLut eLut.cpp )
+-
+-ADD_CUSTOM_COMMAND (
+-  TARGET eLut POST_BUILD
+-  COMMAND eLut > ${CMAKE_CURRENT_BINARY_DIR}/eLut.h
++ADD_CUSTOM_COMMAND(
++  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/eLut.h
++  COMMAND eLut ARGS > ${CMAKE_CURRENT_BINARY_DIR}/eLut.h
++  DEPENDS eLut
+   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+-)
++  )
+ SET_SOURCE_FILES_PROPERTIES(
+   ${CMAKE_CURRENT_BINARY_DIR}/eLut.h
+   PROPERTIES HEADER_FILE_ONLY TRUE
+   )
+ 
+ ADD_EXECUTABLE ( toFloat toFloat.cpp )
+-ADD_CUSTOM_COMMAND (
+-  TARGET toFloat POST_BUILD
+-  COMMAND toFloat > ${CMAKE_CURRENT_BINARY_DIR}/toFloat.h
++ADD_CUSTOM_COMMAND(
++  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/toFloat.h
++  COMMAND toFloat ARGS > ${CMAKE_CURRENT_BINARY_DIR}/toFloat.h
++  DEPENDS toFloat
+   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+-)
++  )
+ SET_SOURCE_FILES_PROPERTIES(
+   ${CMAKE_CURRENT_BINARY_DIR}/toFloat.h
+   PROPERTIES HEADER_FILE_ONLY TRUE
+@@ -27,9 +28,7 @@ SET_SOURCE_FILES_PROPERTIES(
+   half.cpp
+   PROPERTIES
+   OBJECT_DEPENDS
+-  ${CMAKE_CURRENT_BINARY_DIR}/eLut.h
+-  OBJECT_DEPENDS
+-  ${CMAKE_CURRENT_BINARY_DIR}/toFloat.h
++  "${CMAKE_CURRENT_BINARY_DIR}/eLut.h;${CMAKE_CURRENT_BINARY_DIR}/toFloat.h"
+   )
+ 
+ IF(BUILD_SHARED_LIBS)
+diff --git a/Half/half.cpp b/Half/half.cpp
+index 609aaa1..09a50aa 100644
+--- a/Half/half.cpp
++++ b/Half/half.cpp
+@@ -94,9 +94,9 @@ half::convert (int i)
+     // of float and half (127 versus 15).
+     //
+ 
+-    register int s =  (i >> 16) & 0x00008000;
+-    register int e = ((i >> 23) & 0x000000ff) - (127 - 15);
+-    register int m =   i        & 0x007fffff;
++    int s =  (i >> 16) & 0x00008000;
++    int e = ((i >> 23) & 0x000000ff) - (127 - 15);
++    int m =   i        & 0x007fffff;
```

Comment:
I think this should be a different patch since that seems independent of the cmakefiles patches